lib/matplotlib/backends/backend_qt5.py

```diff
@@ -268,7 +268,8 @@ def get_width_height(self):
         return int(w / self._dpi_ratio), int(h / self._dpi_ratio)
 
     def enterEvent(self, event):
-        FigureCanvasBase.enter_notify_event(self, guiEvent=event)
+        x, y = self.mouseEventCoords(event.pos())
+        FigureCanvasBase.enter_notify_event(self, guiEvent=event, xy=(x, y))
```

Comment:
Could do but that would break some inconsistency with the other event as well. For example 

    def wheelEvent(self, event):
        x, y = self.mouseEventCoords(event)
        # from QWheelEvent::delta doc
        if event.pixelDelta().x() == 0 and event.pixelDelta().y() == 0:
            steps = event.angleDelta().y() / 120
        else:
            steps = event.pixelDelta().y()
        if steps:
            FigureCanvasBase.scroll_event(self, x, y, steps, guiEvent=event)

    def keyPressEvent(self, event):
        key = self._get_key(event)
        if key is not None:
            FigureCanvasBase.key_press_event(self, key, guiEvent=event)

    def keyReleaseEvent(self, event):
        key = self._get_key(event)
        if key is not None:
            FigureCanvasBase.key_release_event(self, key, guiEvent=event)

Reply:
Fair.  I suppose those could also all be changed to normal method calls, but that is beyond the scope of this PR.

Reply:
Actually ` self.enter_notif_event(guiEvent=event, xy=(x, y))` does not work